addOns/reports/src/main/zapHomeFiles/reports/high-level-report/report.html

```diff
@@ -42,8 +42,9 @@ <h1>
 		<div class="row">
 			<em
 				th:text="#{report.generated(${#dates.format(new java.util.Date(), 'EEE, d MMM yyyy HH:mm:ss')})}"
-				th:class="'text-right'"></em>
-
+				th:class="'text-right'">Date, time</em> <em
+				th:text="#{report.owaspZapVersion(${zapVersion})}"
+				th:class="'text-right'">OWASP Version</em>
```

Comment:
This should include ZAP.

@psiinon @thc202 should we just use `ZAP Version` for reports? (Ex: without "OWASP").

Reply:
Sounds good.

Reply:
Okay let’s go with just `ZAP Version` throughout.